profile/README.md

```diff
@@ -0,0 +1,22 @@
+![Containers](https://github.com/containers/.github/blob/main/logo/containers-logo.png)
+
+# Welcome to Containers! 
+We're a collection of open source tools that create, configure, and work with containers.
+
+## 🛠️ Tools
+Some of our tools include [Podman](https://github.com/containers/podman), [Podman Desktop](https://github.com/containers/podman-desktop), [Buildah](https://github.com/containers/buildah), [Skopeo](https://github.com/containers/skopeo), [conmon-rs](https://github.com/containers/conmon-rs), and [crun](https://github.com/containers/crun), just to name a few. We also have several supporting libraries for these tools, such as [common](https://github.com/containers/common), [storage](https://github.com/containers/storage), [image](https://github.com/containers/image), and more. Check them out and see all the cool things happening!
+
+## 🤝 Community
+We have an active community. Most of our community communication channels are around [Podman](https://github.com/containers/podman), but we definitely welcome anyone with an interest in containers or any of our tools.
+
+Maintainers of Containers and other community members generally chat here:
+- IRC: #podman on [libera.chat](https://libera.chat/)
+- Matrix: [#podman:matrix.org](https://matrix.to/#/#podman:fedoraproject.org)
+- Discord: [Podman](https://discord.com/invite/x5GzFF6QH4)
```

Comment:
Should we add the Buildah/Podman Twitter channels here?

Reply:
My only concern is if we get requests from a chunk of the other projects for the same, it might get busy.  Perhaps we need a community.md page in each project with this kind of info....   hmm...  Thinking and typing, usually means I find more work for myself!